Src/PCompiler/CompilerCore/CompilerConfiguration.cs

```diff
@@ -53,6 +54,7 @@ public CompilerConfiguration()
             OutputLanguage = outputLanguage;
             Backend = TargetLanguage.GetCodeGenerator(outputLanguage);
             ProjectDependencies = projectDependencies ?? new List<string>();
+            EnableDebugging = false;
```

Comment:
There was a misunderstanding. Instead of a command line flag. I was referring to a build macro that gets set based on the dotnet build debug or release

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives

We would like to use
```
if #DEBUG
```

 then 
```
dotnet -c Release 
or 
dotnet -c Debug
```